### ISSUE TITLE
Add Disney Animation as a Facility

### ISSUE THREAD

**Issue by radford-for-smpte** [ISDCF/registries#999]

**Comment by jackers2024** [comment]
I like the coding schema extension idea better as well.

**Comment by palemieux** [comment]
> Extend the schema for facility codes to be 2-4 characters (which would align with the current studio schema)

No objection from me! Just need to inform the members before we commit.

**Comment by japierce1028** [comment]
I am not in favor of changing the facility code from 3 characters to 4 characters. I'm afraid of breaking something in the workflow of software products. But it's been this way since the beginning (Facility 3 characters, Studio 4 characters) and there are many studios that have an associated facility with a different code. We decided not to require a match a few cycles ago. If we did make the change would it encourage 4-character studios to change their associated facility to match? Bottom line we want to move beyond the naming convention and not keep 'fixing' a convention, not a standard.

**Comment by SteveLLamb** [comment]
Group discussed and ok with adding and expanding the the schema.

**Comment by japierce1028** [comment]
Agreed, we should proceed with allowing 4 character facility codes. I think we should be quiet about it and not list on the website that it is allowed, but change the checks so we can enter a 4 character code in the registry. (I am not committed to not changing the website, but that's my knee-jerk feeling.)

**Comment by jackers2024** [comment]
I agree somewhat, but I think we should wait to reveal the change on the website until that change actually happens.
